json_serializable/test/json_serializable_test.dart

```diff
@@ -365,6 +365,7 @@ String _runForElementNamedWithGenerator(
 final _annotatedElements = _library.allElements
     .map<AnnotatedElement>((e) {
       for (final md in e.metadata) {
+        md.computeConstantValue();
```

Comment:
Zero clue why this changed, but I need to call this or the constant reader doesn't work. 🤷‍♂️

Reply:
Oh... this could actually be an accidental breaking change in build_resolvers. We used to do it for you and maybe we don't any more.

Reply:
FWIW, this is just test infrastructure code. All of the actual code generation worked fine...

Reply:
hmm, I will keep my eye out for similar bug reports but not worry about it for now then